index.html

```diff
@@ -236,24 +259,47 @@ <h1 class="contact_header">
             <input
               type="text"
               name="name"
-              id="name"
+              id="full-name"
               placeholder="Full name"
               minlength="1"
               maxlength="30"
               required
             />
+            <!-- Display none for mobile -->
+            <div class="first-last-name-inputs full-name-container">
+              <input
+                type="text"
+                name="firstname"
+                id="first-name"
+                placeholder="First name"
+                minlength="1"
+                maxlength="30"
+                required
+              />
+              <input
+                type="text"
+                name="lastname"
+                id="last-name"
+                placeholder="Last name"
+                minlength="1"
+                maxlength="30"
+                required
+              />
+            </div>
+            <!-- end of display for mobile -->
```

Comment:
- [ ] Your contact section is good, but there is an important note in the project requirements mentioning that templates that use 'first name' and 'last name' inputs in the desktop version, should be ignored and use only the 'full name' input used in the mobile version. Please remove these inputs and use only the 'full name' input as you used in the mobile version.
 - 
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/61919617/206436154-e163abac-34df-4cc8-959e-41d113a4d47b.png">

Reply:
Thanks for the review